### PR TITLE
Upversion to 2.1.0-SNAPSHOT

### PR DESCRIPTION
**Pull request by rawagner**
Signed-off-by: Rastislav Wagner <rawagner@redhat.com>

### CODE REVIEW

**Comment by rawagner** [comment]
testPR

File: archetype/pom.xml [hunk 1]
```diff
@@ -15,7 +15,7 @@
 	xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>org.eclipse.reddeer</groupId>
-	<version>2.0.1.Final</version>
+	<version>2.0.1.qualifier</version>
```
Comment (odockal) [inline archetype/pom.xml]: @rawagner *2.1.0.qualifier

Comment (rawagner) [reply archetype/pom.xml]: good catch! fixed with latest revision

**Comment by rawagner** [comment]
testPR

**Pull request merged**